packages/rich-text/src/plugins/Table/TableActions.tsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+import { css, cx } from 'emotion';
+import tokens from '@contentful/forma-36-tokens';
+import { SPEditor, useStoreEditor } from '@udecode/slate-plugins-core';
+import {
+  addRow,
+  addColumn,
+  TablePluginOptions,
+  deleteColumn,
+  deleteRow,
+  deleteTable,
+} from '@udecode/slate-plugins-table';
+import {
+  IconButton,
+  Dropdown,
+  DropdownList,
+  DropdownListItem,
+} from '@contentful/forma-36-react-components';
+
+import { addRowAbove, addColumnLeft } from './actions';
+
+export const styles = {
+  topRight: css({
+    position: 'absolute',
+    top: tokens.spacingXs,
+    right: tokens.spacingXs,
+  }),
+};
+
+type TableAction = (e: SPEditor, o: TablePluginOptions) => void;
```

Comment:
Could we use more specific names for `e` and `o`? Not sure what's the best practice but I think it might not be intuitive for future maintenance.

Reply:
I tend to not worry about naming, in short, unexported, _types_ like this. Especially when the arg types tell exactly what to expect (e.g. `SPEditor` and `TablePluginOptions` in this case)

Regardless, I changed them to `editor` and `options`.